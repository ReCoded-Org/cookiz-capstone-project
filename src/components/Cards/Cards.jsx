import Image from "next/image";
const Cards = () => {
    return (
        <div className='mt-20 ml-10 flex flex-col items-center justify-center md:ml-0'>
            {/* title part */}
            <div className='bflex m-auto flex-col'>
                <div className=' flex  flex-col'>
                    <p className=' text-2xl font-bold text-black md:text-3xl'>
                        Youâ€™ve missed home made food?
                    </p>
                    <p className=' pb-12  pt-2 font-normal   text-black'>
                        Pick your favorite chef, food or cousin.
                    </p>
                </div>

                <div className='flex w-full flex-row flex-wrap  justify-between gap-24'>
                    <div className='flex   h-[17.813rem] w-[34.375] flex-row items-center justify-center rounded-[12px] bg-[#9EE2B8] shadow-lg '>
                        <div className='flex flex-col px-6 '>
                            <div>
                                <Image
                                    src='/images/cheff.png'
                                    width={36}
                                    height={36}
                                    layout='intrinsic'
                                    alt='x'
                                />
                            </div>
                            <div className=' h-40 w-[307px] text-2xl font-semibold text-black md:text-3xl'>
                                Pick your chef order the food and get it when
                                you want
                                <div>
                                    <Image
                                        src='/images/soup.png'
                                        width={36}
                                        height={36}
                                        layout='intrinsic'
                                        alt='x'
                                    />
                                </div>
                            </div>
                        </div>
                        <div className='h-[17.8rem] w-full  overflow-hidden rounded-r-lg '>
                            <Image
                                src='/images/chef.png'
                                width={200}
                                height={284}
                                layout='intrinsic'
                                alt='x'
                            />
                        </div>
                    </div>
                    <div className='flex   h-[17.813rem] w-[34.375] flex-row rounded-[12px] bg-[#FFE7E3] shadow-lg '>
                        <div className='flex  flex-col justify-center gap-7 px-6 '>
                            <p className='  w-[310px]   text-2xl   font-semibold text-black md:pt-12 md:text-3xl'>
                                Get a 15% discount for your first order!
                            </p>
                            <button className='  box-border  h-10 w-32 bg-green-900 '>
                                <span className='h-5 w-20  text-base font-bold leading-5 text-white'>
                                    Order Now
                                </span>
                            </button>
                        </div>
                        <div className='h-[17.8rem] w-full  overflow-hidden rounded-r-lg '>
                            <Image
                                src='/images/basket.png'
                                width={200}
                                height={284}
                                layout='intrinsic'
                                alt='x'
                            />
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
};

export default Cards;
