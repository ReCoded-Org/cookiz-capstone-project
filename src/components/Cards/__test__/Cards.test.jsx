import React from "react";
import renderer from "react-test-renderer";

import Cards from "../Cards";

// eslint-disable-next-line
it("renders correctly", () => {
    const tree = renderer.create(<Cards />).toJSON();
    // eslint-disable-next-line no-undef
    expect(tree).toMatchSnapshot();
});
