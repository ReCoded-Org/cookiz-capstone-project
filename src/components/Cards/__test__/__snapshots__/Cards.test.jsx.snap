// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-20 ml-10 flex flex-col items-center justify-center md:ml-0"
>
  <div
    className="bflex m-auto flex-col"
  >
    <div
      className=" flex flex-col"
    >
      <p
        className=" text-2xl font-bold text-black md:text-3xl"
      >
        Youâ€™ve missed home made food?
      </p>
      <p
        className=" pb-12 pt-2 font-normal text-black"
      >
        Pick your favorite chef, food or cousin.
      </p>
    </div>
    <div
      className="flex w-full flex-row flex-wrap justify-between gap-24"
    >
      <div
        className="flex h-[17.813rem] w-[34.375] flex-row items-center justify-center rounded-[12px] bg-[#9EE2B8] shadow-lg "
      >
        <div
          className="flex flex-col px-6 "
        >
          <div>
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "inline-block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "overflow": "hidden",
                  "padding": 0,
                  "position": "relative",
                  "width": "initial",
                }
              }
            >
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              >
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2736%27%20height=%2736%27/%3e"
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                />
              </span>
              <img
                alt="x"
                data-nimg="intrinsic"
                decoding="async"
                onError={[Function]}
                onLoad={[Function]}
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
              <noscript>
                <img
                  alt="x"
                  data-nimg="intrinsic"
                  decoding="async"
                  loading="lazy"
                  src="/_next/image?url=%2Fimages%2Fcheff.png&w=96&q=75"
                  srcSet="/_next/image?url=%2Fimages%2Fcheff.png&w=48&q=75 1x, /_next/image?url=%2Fimages%2Fcheff.png&w=96&q=75 2x"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
              </noscript>
            </span>
          </div>
          <div
            className=" h-40 w-[307px] text-2xl font-semibold text-black md:text-3xl"
          >
            Pick your chef order the food and get it when you want
            <div>
              <span
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "boxSizing": "border-box",
                    "display": "inline-block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "relative",
                    "width": "initial",
                  }
                }
              >
                <span
                  style={
                    Object {
                      "background": "none",
                      "border": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": "initial",
                      "margin": 0,
                      "maxWidth": "100%",
                      "opacity": 1,
                      "padding": 0,
                      "width": "initial",
                    }
                  }
                >
                  <img
                    alt=""
                    aria-hidden={true}
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2736%27%20height=%2736%27/%3e"
                    style={
                      Object {
                        "background": "none",
                        "border": 0,
                        "display": "block",
                        "height": "initial",
                        "margin": 0,
                        "maxWidth": "100%",
                        "opacity": 1,
                        "padding": 0,
                        "width": "initial",
                      }
                    }
                  />
                </span>
                <img
                  alt="x"
                  data-nimg="intrinsic"
                  decoding="async"
                  onError={[Function]}
                  onLoad={[Function]}
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": undefined,
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
                <noscript>
                  <img
                    alt="x"
                    data-nimg="intrinsic"
                    decoding="async"
                    loading="lazy"
                    src="/_next/image?url=%2Fimages%2Fsoup.png&w=96&q=75"
                    srcSet="/_next/image?url=%2Fimages%2Fsoup.png&w=48&q=75 1x, /_next/image?url=%2Fimages%2Fsoup.png&w=96&q=75 2x"
                    style={
                      Object {
                        "border": "none",
                        "bottom": 0,
                        "boxSizing": "border-box",
                        "display": "block",
                        "height": 0,
                        "left": 0,
                        "margin": "auto",
                        "maxHeight": "100%",
                        "maxWidth": "100%",
                        "minHeight": "100%",
                        "minWidth": "100%",
                        "objectFit": undefined,
                        "objectPosition": undefined,
                        "padding": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                        "width": 0,
                      }
                    }
                  />
                </noscript>
              </span>
            </div>
          </div>
        </div>
        <div
          className="h-[17.8rem] w-full overflow-hidden rounded-r-lg "
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27284%27/%3e"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="x"
              data-nimg="intrinsic"
              decoding="async"
              onError={[Function]}
              onLoad={[Function]}
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="x"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2Fchef.png&w=640&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fchef.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fchef.png&w=640&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
      </div>
      <div
        className="flex h-[17.813rem] w-[34.375] flex-row rounded-[12px] bg-[#FFE7E3] shadow-lg "
      >
        <div
          className="flex flex-col justify-center gap-7 px-6 "
        >
          <p
            className=" w-[310px] text-2xl font-semibold text-black md:pt-12 md:text-3xl"
          >
            Get a 15% discount for your first order!
          </p>
          <button
            className=" box-border h-10 w-32 bg-green-900 "
          >
            <span
              className="h-5 w-20 text-base font-bold leading-5 text-white"
            >
              Order Now
            </span>
          </button>
        </div>
        <div
          className="h-[17.8rem] w-full overflow-hidden rounded-r-lg "
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27200%27%20height=%27284%27/%3e"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="x"
              data-nimg="intrinsic"
              decoding="async"
              onError={[Function]}
              onLoad={[Function]}
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="x"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2Fbasket.png&w=640&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fbasket.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fbasket.png&w=640&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
